generators/client/templates/_bower.json

```diff
@@ -34,7 +34,7 @@
     "json3": "3.3.2",
     "modernizr": "3.2.0",
     "ng-file-upload": "10.0.2",
-    "ngInfiniteScroll": "1.2.1",
+    "ngInfiniteScroll": "1.2.2",
```

Comment:
This is to fix a bug that was causing only the first one or two pages to load if the results did not fill the whole screen.